src/open_inwoner/accounts/models.py

```diff
@@ -23,7 +23,14 @@
 
 from .choices import ContactTypeChoices, LoginTypeChoices, StatusChoices, TypeChoices
 from .managers import ActionQueryset, DigidManager, UserManager, eHerkenningManager
-from .query import InviteQuerySet, MessageQuerySet, UserQuerySet
+from .query import InviteQuerySet, MessageQuerySet
+
+
+def generate_uuid_image_name(instance, filename):
+    filename, file_extension = os.path.splitext(filename)
+    return "profile/{uuid}/{file_extension}".format(
```

Comment:
The last separator should be a dot (instead of a slash) so it is still a valid openable file: "profile/{uuid}.{file_extension}"